index.d.ts

```diff
@@ -44,6 +44,8 @@ export type Instance = {
 
 export type Unmount = () => void;
 
+export function useKeypress<T extends (str?: string, key?: any) => null>(keyPresshandler: T): null;
```

Comment:
type for `key`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/de6730f4463cba69904698035fafd906a72b9664/types/node/readline.d.ts#L5-L11

You can import it by `import {Key} from 'readline';`

Should also add a dependency on `@types/node` - I forgot in #1

Reply:
Maybe not the dependency and just hope consumers add it if they need it? IIRC the error from `tsc` is ok (it specifically calls out installing `@types/node`)

The global nature of TS's types seems to cause issues https://github.com/facebook/jest/issues/8092

Reply:
Is the current version okay regarding types? Is there a particular class/book/tutorial you recommend on learning the ins and outs of practical typescript?